src/plugins/condition/components/inspector/ConditionalStylesView.vue

```diff
@@ -1,14 +1,43 @@
 <template>
-<div>Conditional Styles inspector view</div>
+<div class="c-properties__row--span-all">
+    <div class="controls">
+        <div class="preview">ABC</div>
+        <button class="c-icon-button--swatched icon-paint-bucket"
```

Comment:
Can we use v-for here to iterate over a list of properties instead of repeating the button elements?

Reply:
Done